Clinical-Genomics/scout
# Genevariants urldecode hgvs

This PR adds a functionality or fixes a bug.

**How to test**:
1. check a list of gene variants for some variants with VEP HGVS including e.g. synonymous variants like p.Arg101%3D on current master
2. switch to branch and note that they now show URL-decoded like p.Arg101=

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
